builtin/game/chatcommands.lua

```diff
@@ -71,7 +71,7 @@ end
 --
 core.register_chatcommand("me", {
 	params = "<action>",
-	description = "chat action (eg. /me orders a pizza)",
+	description = "Display chat action (eg. /me orders a pizza)",
```

Comment:
Can adjust if you think it's needed, but I thought "chat action" was clear. I think that part shouldn't be removed because it's a very recognisable name for what it is, but perhaps the example could be expanded?  (eg. /me orders a pizza displays '<player name> orders a pizza'). Added 'Display' at the front because all other descriptions are already in the imperative mood and this one wasn't.